pxr/usdImaging/usdImaging/delegate.h

```diff
@@ -659,7 +659,13 @@ class UsdImagingDelegate : public HdSceneDelegate, public TfWeakBase {
     _DependencyMap _dependencyInfo;
 
     void _GatherDependencies(SdfPath const& subtree,
-                             SdfPathVector *affectedCachePaths);
+                             SdfPathVector& affectedCachePaths);
+
+    typedef TfHashMap<SdfPath, SdfPathVector, SdfPath::Hash> _DependenciesCacheMap;
+    _DependenciesCacheMap _dependenciesCacheMap;
```

Comment:
I'm worried about this being confused with _dependencyInfo with the current name.  Some other names that could work: _flattenedDependencyCache, _recursiveDependencyCache, etc.